### PR TITLE
Prepare repo for onboarding

### PR DESCRIPTION
**Pull request by ka7eh**
Most of the files are copy/pasted from other repos. Make sure to look out for typos and references to other projects!

### CODE REVIEW

**Review by maxammann — APPROVED** [review]

**Review by nyurik — APPROVED** [review]
looks good, thx!  One minor thing about code of conduct to make it identical to the gl-js, but otherwise looks good!

File: CODE-OF-CONDUCT.md
Comment (ka7eh) [reply CODE-OF-CONDUCT.md]: Done!

**Comment by nyurik** [comment]
looks good, feel free to merge! Usually i prefer the person who submitted the PR to merge it themselves after someone else approved it -- gives a bit of an ownership and responsibility for submitting :)   Obviously this only applies to those with the merge rights on the repo.

**Pull request merged**
